src/box/space_def.h

```diff
@@ -161,6 +161,8 @@ struct space_def {
 	uint32_t id;
 	/** User id of the creator of the space */
 	uint32_t uid;
+	/** Reference counter. */
+	int refs;
```

Comment:
Introducing reference counting for this structure is an overkill. Better just copy `space_name` to `index_def`.